examples/values-sigstore-openshift.yaml

```diff
@@ -1,259 +1,46 @@
-        name: "fulcio-secret-rh"
-        password: "fulcio"
-        #public_key_file: "keys-cert/file_ca_pub.pem"
-        #private_key_file: "keys-cert/file_ca_key.pem"
-        #root_cert_file: "keys-cert/fulcio-root.pem"
-        public_key: |
-          -----BEGIN PUBLIC KEY-----
-          MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEk52z98mDI8K7Hh5NU0DPZE0v25mO
-          mLRfCAhyvFLWjuAqgWB3V1+ozmH8txXPSjUrmuBA8UybnOkbJNjI64w5UA==
-          -----END PUBLIC KEY-----
-        private_key: |
-          -----BEGIN EC PRIVATE KEY-----
-          Proc-Type: 4,ENCRYPTED
-          DEK-Info: DES-EDE3-CBC,95E8F06458914E40
-
-          Rk2VO4MirGZNBCboN4glwfJy2UAmyuEg/MODiOMa3IWf1DNY2tcpWE4SnUVyAm6L
-          RQaQbUIFMH7iFETyuQPp1jNs+GJWhs4mhwfIYlMCdQLEdAKLPUyCMSQDRaJ+QR2E
-          4u5KAZa/CWkL/HLz226bl4L53eQVW4v0S9apluuhy1s=
-          -----END EC PRIVATE KEY-----
-        root_cert: |
-          -----BEGIN CERTIFICATE-----
-          MIICtzCCAlygAwIBAgIJAME0VsLfyjElMAoGCCqGSM49BAMCMHIxCzAJBgNVBAYT
-          AlVTMRcwFQYDVQQIDA5Ob3J0aCBDYXJvbGluYTEQMA4GA1UEBwwHUmFsZWlnaDEQ
-          MA4GA1UECgwHUmVkIEhhdDEmMCQGA1UECwwdUmVkIEhhdCBFbWVyZ2luZyBUZWNo
-          bm9sb2dpZXMwHhcNMjMwNzMxMTEwODE2WhcNMjQwNzMwMTEwODE2WjByMQswCQYD
-          VQQGEwJVUzEXMBUGA1UECAwOTm9ydGggQ2Fyb2xpbmExEDAOBgNVBAcMB1JhbGVp
-          Z2gxEDAOBgNVBAoMB1JlZCBIYXQxJjAkBgNVBAsMHVJlZCBIYXQgRW1lcmdpbmcg
-          VGVjaG5vbG9naWVzMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEk52z98mDI8K7
-          Hh5NU0DPZE0v25mOmLRfCAhyvFLWjuAqgWB3V1+ozmH8txXPSjUrmuBA8UybnOkb
-          JNjI64w5UKOB2jCB1zAPBgNVHRMBAf8EBTADAQH/MB0GA1UdDgQWBBTjoQToDWhE
-          C+yS2X4pIEB8jqWsCDCBpAYDVR0jBIGcMIGZgBTjoQToDWhEC+yS2X4pIEB8jqWs
-          CKF2pHQwcjELMAkGA1UEBhMCVVMxFzAVBgNVBAgMDk5vcnRoIENhcm9saW5hMRAw
-          DgYDVQQHDAdSYWxlaWdoMRAwDgYDVQQKDAdSZWQgSGF0MSYwJAYDVQQLDB1SZWQg
-          SGF0IEVtZXJnaW5nIFRlY2hub2xvZ2llc4IJAME0VsLfyjElMAoGCCqGSM49BAMC
-          A0kAMEYCIQCS9BVRLPjGhFEJPc4mMD2g8lc275d+OO4qx/OswysyigIhAPzcJCf8
-          szmI+ON/YItgbv0e5Wu7t9700ujBNfgMhzyO
-          -----END CERTIFICATE-----
-
   cosign:
-    BASE_DOMAIN: apps.<OPENSHIFT_BASE_DOMAIN>
+    BASE_DOMAIN: <OPENSHIFT_BASE_DOMAIN>
+  fulcio:
+    create: false
+  rekor:
+    create: false
 
+# github.com/sigstore/helm-charts/charts
 scaffold:
   fulcio:
-    namespace:
-      name: fulcio-system
-      create: false
-    createcerts:
-      enabled: false
-      image:
-        registry: quay.io
-        repository: securesign/createcerts
-        version: v0.6.4
-        pullPolicy: IfNotPresent
     server:
-      image:
-        registry: quay.io
-        repository: securesign/fulcio
-        version: v1.4.0
-        pullPolicy: IfNotPresent
-      secret: fulcio-secret-rh
       ingress:
         http:
-          annotations:
-            route.openshift.io/termination: "edge"
-          className: ""
           hosts:
-            - host: fulcio.apps.<OPENSHIFT_BASE_DOMAIN>
+            - host: fulcio.<OPENSHIFT_BASE_DOMAIN>
```

Comment:
See comment related to the format of the parameters. Should the format be modified to take in an environment variable?

Reply:
yes, agreed, updated!